redbot/core/core_commands.py

```diff
@@ -1487,7 +1470,7 @@ async def invite(self, ctx):
             - `[p]invite`
         """
         try:
-            await ctx.author.send(await self._invite_url())
+            await ctx.author.send(await self.bot.get_invite_url)
```

Comment:
```suggestion
            await ctx.author.send(await self.bot.get_invite_url())
```

Reply:
I don't see this registered as a property - so unless im missing something this still needs to be a function call.

Reply:
This indeed needs to be changed.

Reply:
Thank you